src-vue/src/plugins/store.ts

```diff
@@ -61,8 +60,7 @@ export const store = createStore<FlightCoreStore>({
     state(): FlightCoreStore {
         return {
             developer_mode: false,
-            game_path: undefined as unknown as string,
-            install_type: undefined as unknown as InstallType,
+            game_install: undefined as unknown as GameInstall,
```

Comment:
We may want to use an object with default values to prevent any race condition during development.

Reply:
Given that it worked fine before moving the two values into an object I think it should be fine ^^

Reply:
Yeah but now we are trying to access members of `undefined`, resulting in this  error:
`Unhandled Promise Rejection: TypeError: undefined is not an object (evaluating '_ctx.$store.state.game_install.game_path')`

Turning on my Web Dev brain for a moment, just using an empty object should be enough since all uninitialized members will become unknown.